packages/playground-ui/src/ds/components/Logo/logo.css

```
@@ -0,0 +1,63 @@
+
+.logo-animate-on-hover .logo-stroke {
+  animation: logo-stroke-cycle 1500ms cubic-bezier(0.4, 0, 0.2, 1) 1;
+}
+
+@keyframes logo-fill-cycle {
+  0% {
+    fill-opacity: 1;
+  }
+  23.3% {
+    fill-opacity: 0;
+  }
```

Stylelint (17.7.0)

[error] Expected empty line before rule (rule-empty-line-before)

(rule-empty-line-before)